internal/plugins/helm/v1/init.go

```diff
@@ -160,6 +162,13 @@ func (p *initPlugin) runPhase2() error {
 	if err := scorecard.RunInit(p.config); err != nil {
 		return err
 	}
+
+	if p.doCreateAPI {
+		if err := p.apiPlugin.runPhase2(); err != nil {
```

Comment:
What means `runPhase2`?
Also, IHMO if it is post customisation that should be done just when the API is a scaffold then, it should be in the CreateAPI plugin instead of the init one. The init would call the create API plugin.